app/grandchallenge/reader_studies/views.py

```diff
@@ -1274,3 +1254,88 @@ class QuestionInterfacesView(View):
     def get(self, request):
         form = QuestionForm(request.GET)
         return HttpResponse(form["interface"])
+
+
+class DisplaySetDetail(
+    LoginRequiredMixin, ObjectPermissionRequiredMixin, DetailView
+):
+    template_name = "reader_studies/readerstudy_display_set_detail.html"
+    model = DisplaySet
+    permission_required = (
+        f"{ReaderStudy._meta.app_label}.view_{DisplaySet._meta.model_name}"
+    )
+    raise_exception = True
+
+
+class DisplaySetUpdate(
+    LoginRequiredMixin,
+    ObjectPermissionRequiredMixin,
+    UpdateView,
+):
+    template_name = "reader_studies/readerstudy_display_set_update.html"
+    model = DisplaySet
+    form_class = DisplaySetForm
+    permission_required = (
+        f"{ReaderStudy._meta.app_label}.change_{DisplaySet._meta.model_name}"
+    )
+    raise_exception = True
+
+    def form_valid(self, form):
```

Comment:
Shouldn't this all be in the forms save method? Isn't the call to `super().form_valid()` missing?

Reply:
Could you add a comment in the code there, important for future users to know why.

I think it should redirect, this always seems like a fairly fundamental thing for web development (https://en.wikipedia.org/wiki/Post/Redirect/Get).